README.rst

```diff
@@ -49,7 +49,7 @@ Workspace CLI Examples
 The implemented commands for the Workspace CLI can be listed by running ``databricks workspace -h``.
 Commands are run by appending them to ``databricks workspace``. To make it easier to use the workspace
 CLI, feel free to alias ``databricks workspace`` to something shorter. For more information
-reference `Aliasing Command Groups <#aliasing-command-groups>`_.
+reference `Aliasing Command Groups section <#aliasing-command-groups>`_.
```

Comment:
Need to change the text because the section heading is using the same link text. Tried `:ref:` but it seems GitHub doesn't render it.